cpp/models/ide_secir/parameters_io.cpp

```diff
@@ -183,10 +232,10 @@ IOResult<void> set_initial_flows(Model& model, ScalarType dt, std::string const&
     }
 
     //--- Calculate the remaining flows. ---
-    // Compute flow ExposedToInfectedNoSymptoms for -global_support_max, ..., 0.
+    // Compute flow ExposedToInfectedNoSymptoms for -2 * global_support_max, ..., 0.
```

Comment:
Why is this necessary?

Reply:
Ah to calculate the InfectedNoSymptomsToRecovered transition, right?

Reply:
Yes